internal/controller/standalone_pgadmin/users.go

```diff
@@ -0,0 +1,286 @@
+// Copyright 2023 - 2024 Crunchy Data Solutions, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package standalone_pgadmin
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/crunchydata/postgres-operator/internal/logging"
+	"github.com/crunchydata/postgres-operator/internal/naming"
+	"github.com/crunchydata/postgres-operator/internal/util"
+	"github.com/crunchydata/postgres-operator/pkg/apis/postgres-operator.crunchydata.com/v1beta1"
+)
+
+type Executor func(
+	ctx context.Context, stdin io.Reader, stdout, stderr io.Writer, command ...string,
+) error
+
+type secretPGAdminUser struct {
```

Comment:
A comment on this talking about where these values come from would be helpful.

I also think this could just be `pgAdminUser`, I'm not sure what the `secret` descriptor is adding

Reply:
The `secret` part is there because this struct is used for the user data that ends up in the `users.json` file in our secret, and differentiates it from the `PGAdminUser` struct that we already have in `standalone_pgadmin_types.go` that is used in the spec... I'm open to renaming it though if you want.

Reply:
ah, I see. maybe `pgAdminUserJson` :thinking: